tests/PHPStan/Rules/PhpDoc/data/value-of-enum.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace ValueOfEnum;
+namespace ValueOfEnumPhpdoc;
```

Comment:
this namespace collides with the one in `tests/PHPStan/Analyser/data/value-of-enum.php` and therefore we get same-class errors when running tests in a single process